app/src/main/java/org/eyeseetea/malariacare/domain/entity/Question.java

```diff
@@ -447,15 +438,15 @@ public String getValue() {
                 return value;
             }
 
-            public void setValue(String value) {
+            private void setValue(String value) {
```

Comment:
Can you remove private setters? It's not necessary we have private setters, so we reduce the size of the class

Reply:
Done!